### PR TITLE
Fixed deepcopy generation

### PR DESCRIPTION
**Pull request by inercia**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -131,8 +154,11 @@ fmt: $(REGS_OPER_SRCS)
 simplify:
 	@gofmt -s -l -w $(REGS_OPER_SRCS)
 
+$(GOLINT):
+	-@$(GO_NOMOD) get golang.org/x/lint/golint
+
 .PHONY: check
-check:
+check: $(GOLINT)
 	@test -z $(shell gofmt -l $(REGS_OPER_MAIN) | tee /dev/stderr) || echo "[WARN] Fix formatting issues with 'make fmt'"
```
Comment (MalloZup) [inline Makefile]: we can add also here the go vet 
https://github.com/kubic-project/kubic-init/blob/master/Makefile#L131

Comment (MalloZup) [reply Makefile]: :cake:

Comment (inercia) [reply Makefile]: Yeah, we could port some of these things to the `kubic-init` Makefile too...

Comment (MalloZup) [reply Makefile]: yop i think we should try to keep them same pattern as we can or it is possible more or less :cactus:

**Pull request merged**